### DISCUSSION TITLE
IMPORTANT: Project goals 2022 for turf_dart

### DISCUSSION THREAD

**Discussion by lukas-h** [dartclub/turf_dart#59] [Announcements]
Dear fellow maintainers and contributors,

@baparham @tobrun and others who are interested to join,

**I would really like to make a short video meeting about turf_dart to discuss some milestones and overall goals for this year!**
I am hiring a full time dev who will dedicate half of his time for the next few months to this library and following some months later, I want to hire other staff to work on it too.

I really appreciate your contributions and would like get your input on:
- which features we should prioritize
- what you think is important or what you need from the library / want to see implemented first / your use cases
- some overall design decisions
etc...

Let me know if and when you have time for that and I'll schedule the meeting!

(I already sent this to some people via email!)

**Comment by lukas-h** [comment]
My email address is lukas (at) scalabs.de, if you want to reach out privately

**Comment by lukas-h** [comment]
Let’s gather diecussion points for the meeting. Mine include:
#46 
#58 
#54
#12 porting all features of the meta package in Q1 2022, is no.1 priority

**Comment by lukas-h** [comment]
CC @armantorkzaban

**Comment by lukas-h** [comment]
We talked about different topics and came to some conclusions:

- https://github.com/dartclub/turf_dart/issues/46#issuecomment-1057465724
- I created some milestones, following our prioritization: https://github.com/dartclub/turf_dart/milestones

more follows shortly

**Reply by armantorkzaban** [reply]
I missed the last meeting, but now that we have made some progress I hope we can meet again and discuss the next steps together.
